packages/InteractiveProfilingTool-Core.package/IPTModel.class/instance/buildButtonWith..st

```diff
@@ -0,0 +1,20 @@
+user interface
+buildButtonWith: aBuilder
+	"Button that runs toggleProfiling when clicked with variable label."
```

Comment:
> While I’m personally in favor of more comments instead of less, the majority opinion in the Squeak community seems to be to treat comments as an exception… So I guess the people grading us would consider this comment redundant.

Same applies here.